src/app/blog/components/post-detail/post-detail.component.html

```diff
@@ -61,7 +62,7 @@ <h1 data-aos="fade-in-up" data-aos-offset="100" class="text-primary">BLOG</h1>
         <div class="p-field p-grid">
           <div class="p-inputgroup">
             <span class="p-inputgroup-addon"><i class="pi pi-envelope"></i></span>
-            <input pInputText formControlName="email" type="email" placeholder="Email">
+            <input pInputText formControlName="email" type="email" placeholder="Email *">
```

Comment:
I read a this artile last time https://angular.io/api/forms/EmailValidator
Can you use it here : It shows how to valid an email properly
Only chage is in HTML file 

I understand from that article : We need to use **attribut email** in input element : SO EASY

Reply:
yes we just have to use type="email"